### PR TITLE
added notebook for uploading model data

### PR DESCRIPTION
**Pull request by tomw66**
I've added a notebook to help with uploading AQUM model data, similar to the one for MOASA flight data.

### CODE REVIEW

File: helper-funcs/uploading_AQUM_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Discover the Data\n",
+    "`anon=False` means you will have to set up your access credentials before being able to get the data - see the notebook 'access_datastore' for help."
+   ]
```
Comment (corinnebosley) [inline helper-funcs/uploading_AQUM_data.ipynb:40]: Is there something we can use here for access credentials, just to demonstrate the process, like a dummy log-in or something?

If not, this is fine, but it means that I can't actually run this cell or anything following it as I don't have access.  I can of course run the `access_datastore` notebook to get them (I assume), so I'll do that now.

File: helper-funcs/uploading_AQUM_data.ipynb [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Download the Data\n",
+    "These files are pretty big, so we're only downloading those that aren't already in our local `temp_holder/` directory, or our `caf-data` store. \n",
+    "\n",
+    "Use `number_of_files` to specify how many files you want to download at a time, rather than doing them all at once."
+   ]
```
Comment (corinnebosley) [inline helper-funcs/uploading_AQUM_data.ipynb:61]: I like the clarity here.

**Review by corinnebosley — CHANGES_REQUESTED** [review]
One or two small changes in the text, with an option for using a dummy credential if possible.

Other than that, looks like a lovely notebook.  Nice work.

File: helper-funcs/uploading_AQUM_data.ipynb (hunk 1 above)
Comment (tomw66) [reply helper-funcs/uploading_AQUM_data.ipynb:40]: I don't think so, would maybe be a security issue for jasmin. It's probably a good idea for you to get credentials set up anyway, though it is fiddly so let me know if you need help!

**Review by corinnebosley — APPROVED** [review]

File: helper-funcs/uploading_AQUM_data.ipynb (hunk 1 above)
Comment (corinnebosley) [reply helper-funcs/uploading_AQUM_data.ipynb:41]: You're not wrong.  I'll get my shizzle together and get access to JASMIN myself.

**Comment by corinnebosley** [comment]
Nice notebook Tom.  Merge when ready.

**Pull request merged**